nuxeo-features/nuxeo-platform-comment/nuxeo-platform-comment/src/main/java/org/nuxeo/ecm/platform/comment/impl/PropertyCommentManager.java

```diff
@@ -451,4 +455,16 @@ protected DocumentModel getThreadForComment(CoreSession s, DocumentModel comment
             return documentModel;
         });
     }
+
+    @Override
+    public DocumentRef getCommentedDocument(CoreSession s, DocumentModel commentDocumentModel) {
+        return CoreInstance.doPrivileged(s, session -> {
+            String commentedDocId = commentDocumentModel.getId();
+            if (commentDocumentModel.hasSchema(COMMENT_SCHEMA)) {
+                commentedDocId = (String) commentDocumentModel.getPropertyValue(COMMENT_PARENT_ID);
+            }
```

Comment:
It seems that we now tend to use ternary, WDYT?

Reply:
why not, mainly i used it (the ternary operator) to  set a variable value.
rewrite it as: 
```
String commentedDocId = commentDocumentModel.hasSchema(COMMENT_SCHEMA)
                    ? (String) commentDocumentModel.getPropertyValue(COMMENT_PARENT_ID)
                    : commentDocumentModel.getId();
```
seem to me that don't reduce code enough

Reply:
I like it this way, but again, we still don't have clear conventions about this, so, as you wish :smile: